pymc3/distributions/multivariate.py

```diff
@@ -2098,3 +2098,62 @@ def logp(value, mu, W, alpha, tau):
             at.all(alpha >= -1),
             tau > 0,
         )
+
+
+class StickBreakingWeightsRV(RandomVariable):
+    name = "stick_breaking_weights"
+    ndim_supp = 1
+    ndims_params = [0]
```

Comment:
Following and revisiting the discussion [here](https://github.com/larryshamalama/pymc3-playground/pull/1) with @Sayam753 , my current guess is that `ndim_supp = 1` and `ndims_params = [0]` because `alpha` should be a scalar and the sampled weights consist of a 1D vector. Still not 100% sure of this.

Reply:
Seems correct